app/src/main/java/com/example/sofascoreapp/viewmodel/SpecificTournamentViewModel.kt

```diff
@@ -0,0 +1,89 @@
+package com.example.sofascoreapp.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.cachedIn
+import androidx.paging.liveData
+import com.example.sofascoreapp.data.model.Standing
+import com.example.sofascoreapp.data.model.Tournament
+import com.example.sofascoreapp.data.networking.Network
+import com.example.sofascoreapp.ui.paging.TeamEventsPagingSource
+import com.example.sofascoreapp.ui.paging.TournamentEventsPagingSource
+import kotlinx.coroutines.async
+import kotlinx.coroutines.launch
+import retrofit2.Response
+
+class SpecificTournamentViewModel : ViewModel() {
+
+    private val _tournamentID = MutableLiveData<Int>()
+
+    fun setTournamentID(id: Int) {
+        _tournamentID.value = id
+    }
+
+    fun getTournamentID(): MutableLiveData<Int> {
+        return _tournamentID
+    }
+
+    private val _tournamentStandings = MutableLiveData<Response<ArrayList<Standing>>>()
+
+    fun setTournamentStandings(standings: Response<ArrayList<Standing>>) {
+        _tournamentStandings.value = standings
+    }
+
+    fun getTournamentStandings(): MutableLiveData<Response<ArrayList<Standing>>> {
```

Comment:
kad vracas van liveDatu, trebao bi ju vratiti kao immutable live datu jer ne zelis da ju itko mijenja izvana
```suggestion
    fun getTournamentStandings(): LiveData<Response<ArrayList<Standing>>> {
```